### DISCUSSION TITLE
🔥可以用一句话描述你心中的web3是什么吗？你为什么选择从事web3领域呢？举一个你对于web3印象最深的一个案例/事件？

### DISCUSSION THREAD

**Discussion by luuuuo** [Ed3Academy/blockchain-courses#6] [随便聊聊]
当你能发现这个链接并点进来的时候相信你已经对web3.0开始感兴趣了，欢迎入坑！🚑
> Web 3.0 是结合了去中心化和通证（Token）经济学等概念，基于区块链技术的全新的互联网迭代方向。
> --https://zh.wikipedia.org/wiki/Web3

🚨可以用一句话描述你心中的web3是什么吗？
🔥你为什么选择从事web3领域呢？
🐳举一个你对于web3印象最深的一个案例/事件？

**Comment by Huanbinbin16** [comment]
🚨可以用一句话描述你心中的web3是什么吗？---自由 公平
🔥你为什么选择从事web3领域呢？--组织安排
🐳举一个你对于web3印象最深的一个案例/事件？--1w比特币兑换一个披萨

**Reply by luuuuo** [reply]
Are you kidding? No I'm serious.
你是鸡丁先生吗？不，我是谢尔瑞思

**Comment by Huanbinbin16** [comment]
这是一封自动回复邮件。已经收到您的来信，我会尽快回复。
